### PR TITLE
Add version to PowerShell banner

### LINKED ISSUE #3895: Include GitCommitId in the console banner
Steps to reproduce
------------------
```
powershell
```

Expected behavior
-----------------
```
PowerShell v6.0.0-beta.1
Copyright (C) Microsoft Corporation. All rights reserved.

PS /home/steve>
```
Since we support side-by-side, it makes it easier to see which one you started rather than checking $psversiontable or $pshome (in the case of whether you're in an official release or one you just built)

Actual behavior
---------------
```
PowerShell 
Copyright (C) Microsoft Corporation. All rights reserved.

PS /home/steve>
```

Environment data
----------------
```powershell
Name                           Value
----                           -----
PSVersion                      6.0.0-beta
PSEdition                      Core
BuildVersion                   3.0.0.0
CLRVersion
GitCommitId                    v6.0.0-beta.1
OS                             Microsoft Windows 10.0.16197
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### PR DESCRIPTION
**Pull request by iSazonov**
Close #3895 

Motivation
---
Since we support side-by-side, it makes it easier to see which PowerShell version you started rather than checking $psversiontable or $pshome.

Fix
---

Add "GitCommitId" to PowerShell banner.

Follow-up work
---
Based on #3690 later we will change a GitCommitId format.

### CODE REVIEW

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx [hunk 1]
```diff
@@ -119,11 +119,13 @@
...
     <value>Windows PowerShell 
-Copyright (C) Microsoft Corporation. All rights reserved.</value>
+Copyright (C) Microsoft Corporation. All rights reserved.
+Version: {0}</value>
   </data>
   <data name="ShellBannerNonWindowsPowerShell" xml:space="preserve">
     <value>PowerShell 
-Copyright (C) Microsoft Corporation. All rights reserved.</value>
+Copyright (C) Microsoft Corporation. All rights reserved.
+Version: {0}</value>
```
Comment (vors) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: I think the version proposed by @SteveL-MSFT originally reads nicer:

```
PowerShell {0}
Copyright (C) Microsoft Corporation. All rights reserved.
```

For example, here is how python banner looks like
```
Python 2.7.10 (default, Oct 18 2015, 00:31:39) 
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.0.72)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
```

**Review by vors — CHANGES_REQUESTED** [review]

File: src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx (hunk 1 above)
Comment (iSazonov) [inline src/Microsoft.PowerShell.ConsoleHost/resources/ManagedEntranceStrings.resx]: I think it doesn't look elegant :-) but fixed.

**Review by SteveL-MSFT — APPROVED** [review]
LGTM

**Comment by iSazonov** [comment]
@vors Can you approve and merge?

**Pull request merged**